src/overlays/actors/ovl_En_Ishi/z_en_ishi.c

```diff
@@ -370,66 +391,67 @@ s32 EnIshi_IsUnderwater(EnIshi* this, PlayState* play) {
                                 &waterBox, &bgId) &&
         (this->actor.world.pos.y < waterSurface)) {
         return true;
+    } else {
+        return false;
     }
-    return false;
 }
 
 void EnIshi_Init(Actor* thisx, PlayState* play) {
     s32 pad;
     EnIshi* this = (EnIshi*)thisx;
-    s32 sp34 = ENISHI_GET_1(&this->actor);
-    s32 sp30 = ENISHI_GET_4(&this->actor);
+    s32 isBig = ENISHI_GET_BIG_FLAG(&this->actor);
+    s32 ignoreSnapToFloor = ENISHI_GET_IGNORE_SNAP_TO_FLOOR(&this->actor);
 
-    if ((sp34 == 0) && (sp30 != 0)) {
-        this->unk_197 |= 2;
+    if ((isBig == false) && (ignoreSnapToFloor != false)) {
```

Comment:
```suggestion
    if (!isBig && ignoreSnapToFloor) {
```

Reply:
I know there is only two options, but I am actually wondering if we should actually treat `isBig` more as an enum rather than a booelan, especially since it gets used as an index into an array.